lottie/src/main/java/com/airbnb/lottie/utils/LottieValueAnimator.java

```diff
@@ -121,7 +121,22 @@ private float getFrameDurationNs() {
 
   public void setComposition(LottieComposition composition) {
     this.composition = composition;
-    setMinAndMaxFrames((int) composition.getStartFrame(), (int) composition.getEndFrame());
+
+    int minFrame;
+    if (this.minFrame != Integer.MIN_VALUE) {
```

Comment:
@Kevinrob Can't you just use the Math.min and math.max case for both? If the minFrame or maxFrame are MIN_VALUE or MAX_VALUE then it should use the start frame or end frame so the result would be the same as the else branch right?

Reply:
You are right! I changed it 👍